src/service/video-manager-impl.js

```diff
@@ -318,7 +318,7 @@ export class VideoManager {
    * with it or playing through autoplay
    *
    * @param {!../video-interface.VideoInterface} video
-   * @return {!../video-interface.VideoInterface} PlayingStates
+   * @return {string}
```

Comment:
`PlayingStates` sounds far more descriptive than `string`, @alanorozco does this look correct?

Reply:
I'm glad you're keeping me honest here. The return type as originally written seemed meaningless. Probably a copy-pasta typo. Without this change:
```js
src/service/video-manager-impl.js:324: ERROR - inconsistent return type
found   : string
required: VideoInterface$$module$src$video_interface
    return this.getEntryForVideo_(video).getPlayingState();
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

src/service/video-manager-impl.js:1207: ERROR - inconsistent return type
found   : VideoInterface$$module$src$video_interface
required: string
    return this.manager_.getPlayingState(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```
Unfortunately, using `PlayingStates` (or the fully qualified `!../video-interface.PlayingStates`) as a return type does not work, perhaps because it is a mapping of `string` to `string`, while this function returns just a `string`. Errors seen:

```js
src/service/video-manager-impl.js:321: ERROR - Bad type annotation. Unknown type PlayingStates$$module$src$video_interface
   * @return {!../video-interface.PlayingStates}
```
or
```js
src/service/video-manager-impl.js:321: ERROR - Bad type annotation. Unknown type PlayingStates$$module$src$video_interface
   * @return {PlayingStates}
```

Reply:
The `VideoInterface` type should probably be fixed instead. I would prefer if we don't lose this type information here.

Reply:
@alanorozco Do you have an alternative in mind? (See comment above for my failed attempts at using `PlayingStates` as a return type.)

Reply:
That worked. Done in https://github.com/ampproject/amphtml/commit/919c7fbd4143768cb1b14e18ed46c2158ff0a44b.